packages/zosfiles/src/api/methods/utilities/Utilities.ts

```diff
@@ -111,4 +111,23 @@ export class Utilities {
         }
         return false;
     }
+
+    /**
+     * Re-name USS file or directory
+     *
+     * @param {AbstractSession} session     - z/OSMF connection info
+     * @param {string}          USSFilePath - contains the current filepath
+     * @param {string}          newFilePath - contains the new filepath
+     *
+     * @returns {Promise<Buffer>} Promise that resolves to json information
+     *
+     * @throws {ImperativeError}
+     */
+    public static async renameUSSFile(session: AbstractSession, USSFilePath: string, newFilePath: string): Promise<Buffer> {
+        ImperativeExpect.toNotBeNullOrUndefined(newFilePath, ZosFilesMessages.missingUSSFileName.message);
+        const oldFilePath = USSFilePath.charAt(0) === "/" ? USSFilePath : "/" + USSFilePath;
+        const payload = { request: "move",  from: path.posix.normalize(oldFilePath) };
```

Comment:
`ZosFilesUtils.sanitizeUssPathForRestCall` removes the first forward slash in the path and URI encodes the path. Both of these things break the functionality.

Reply:
OK. Makes sense